src/scripts/main.js

```diff
@@ -1,7 +1,35 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  const body = document.querySelector("body")
+  const message = document.createElement("div")
+  message.classList.add('notification')
+  message.classList.add(type);
+
+
+  const titleElement = document.createElement('h2')
+  titleElement.classList.add('title')
+  titleElement.textContent = title;
+  message.appendChild(titleElement)
+
+
+  const descriptionElement = document.createElement('p')
+  descriptionElement.textContent = description;
+  message.appendChild(descriptionElement)
+
+  message.style.top = `${posTop}px`;
+  message.style.right = `${posRight}px`;
+
+  body.appendChild(message)
+
+  setTimeout(() => {
+    message.remove()
+
+  }, 2000)
+
+
+
+
 };
```

Comment:
The `pushNotification` function is doing too many things: creating elements, adding classes, setting styles, and appending to the body. This violates the Single Responsibility Principle. Consider breaking it up into smaller, more focused functions, each doing one thing. For example, you could have separate functions for creating a notification, setting its content, and displaying it.